src/rez_pip/rez.py

```diff
@@ -113,8 +136,8 @@ def make_root(variant: rez.packages.Variant, path: str) -> None:
         pkg.pip = {
             "name": dist.name,
             "version": dist.version,
-            "is_pure_python": metadata["is_pure_python"],
-            "wheel_url": wheelURL,
+            "is_pure_python": isPure,
```

Comment:
This annoys me. I don't think we really need this...

Reply:
It's kind of useless and getting the value of "is pure" is annoying. Do you think I should keep it?